### PR TITLE
Ensure that the branding packages are installed in the containers

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: tests/test_all.py [hunk 1]
```diff
@@ -88,6 +88,20 @@ def test_os_release(auto_container):
         )
 
 
+def test_branding(auto_container):
+    """
+    check that the :file:`/etc/SUSE-brand` file exists and containers SLE branding
+    """
+    assert auto_container.connection.file("/etc/SUSE-brand").exists
```
Comment (dcermak) [inline tests/test_all.py:95]: The location is `/usr/etc/SUSE-brand` on Tumbleweed.

**Review by dcermak — COMMENTED** [review]
Do we really want to have these packages in the container? I really don't see the value add of a file with the contents:
```
SLE
VERSION = 15
```
That's just a worse `/etc/os-release`.

File: tests/test_all.py (hunk 1 above)
Comment (dirkmueller) [reply tests/test_all.py:95]: gah

File: tests/test_all.py [hunk 2]
```diff
@@ -89,6 +89,29 @@ def test_os_release(auto_container):
...
+    reason="branding packages are known to not be installed",
+)
+@pytest.mark.parametrize(
+    "container",
+    CONTAINERS_WITH_ZYPPER,
+    indirect=True,
+)
+def test_branding(container):
+    """
+    check that the :file:`/etc/SUSE-brand` file exists and containers SLE branding
```
Comment (dcermak) [inline tests/test_all.py:103]:
```suggestion
    check that the :file:`/etc/SUSE-brand` file exists and contains the SLE branding
```

File: tests/test_all.py [hunk 3]
```diff
@@ -89,6 +89,29 @@ def test_os_release(auto_container):
...
+    """
+    location = "/etc/SUSE-brand"
+    branding = "SLE"
+    if OS_VERSION == "tumbleweed":
+        branding = "openSUSE"
+    if OS_VERSION in ("basalt", "tumbleweed"):
+        location = "/usr/etc/SUSE-brand"
```
Comment (dcermak) [inline tests/test_all.py:110]: Why use these switches when the tests only run on Tumbleweed?

Comment (dirkmueller) [reply tests/test_all.py:110]: it also runs on 15.6 (and it did until recently basalt completely exploded) also on basalt. eventually I hope to be able to fix it also for older versions, so the `skipif` will reduce over time (famous last words)

**Pull request merged**